docs/release_notes/v1.0.5.md

```diff
@@ -0,0 +1,98 @@
+# Spice v1.0.5 (Mar 10, 2025)
+
+Spice v1.0.5 expands Iceberg support with the introduction of the **Iceberg Data Connector**, in addition to the existing Iceberg Catalog Connector. This new connector enables direct dataset creation and configuration for specific Iceberg objects, enabling federated and accelerated SQL queries on [Apache Iceberg](https://iceberg.apache.org/) tables.
+
+Performance improvements include enhanced Parquet pruning in append mode, where object-store metadata is now leveraged alongside Hive partitioning to optimize file pruning. This results in faster and more efficient queries.
+
+DuckDB has been upgraded to v1.2.0, along with additional stability improvements, including improved graceful shutdown.
+
+Additional updates include support for the Arrow Map type.
+
+## Highlights in v1.0.5
+
+- **New Iceberg Data Connector**: Enables direct dataset creation and querying of Iceberg tables.
+
+Example usage in `spicepod.yaml`:
+
+```yaml
+datasets:
+  - from: iceberg:https://iceberg-catalog-host.com/v1/namespaces/my_namespace/tables/my_table
+    name: my_table
+    params:
+      # Same as Iceberg Catalog Connector
+    acceleration:
+      enabled: true
+```
+
+For detailed setup instructions, authentication options, and configuration parameters, refer to the [Iceberg Data Connector documentation](https://spiceai.org/docs/components/data-connectors/iceberg).
+
+- **Improved Parquet pruning in append mode**: Uses object-store metadata for more efficient file pruning.
+
+- **DuckDB upgrade to v1.0.2 with improved graceful shutdown**: Read the [DuckDB v1.2.0 announcement](https://duckdb.org/2025/02/05/announcing-duckdb-120.html) for details, including breaking changes for `map` and `list_reduce`. Graceful shutdown of DuckDB has been improved for better stability across restarts.
+
+- **Arrow Map type support**: Expands compatibility with Arrow-based workflows.
```

Comment:
```suggestion
- **Configure DuckDB memory limit**: Use `duckdb_memory_limit` parameter to set the DuckDB acceleration memory limit:

```yaml
- from: spice.ai:path.to.my_dataset
   name: my_dataset
   acceleration:
     params: 
       duckdb_memory_limit: '2GB'
     enabled: true
     engine: duckdb
     mode: file
```

- **Arrow Map type support**: Expands compatibility with Arrow-based workflows.
```